ASM/c/item_table.c

```diff
@@ -327,6 +327,21 @@ item_row_t item_table[] = {
     [0x0116] = ITEM_ROW(0x53,      SILVER_CHEST, 0x41, -1, 0x00A0, 0x01A0, 0x87, no_upgrade, give_small_key, FORT_ID,   -1, resolve_text_small_keys), // Thieves' Hideout Small Key
     [0x0117] = ITEM_ROW(0x53,      SILVER_CHEST, 0x41, -1, 0x00A1, 0x01A1, 0x88, no_upgrade, give_small_key, CASTLE_ID, -1, resolve_text_small_keys), // Ganon's Castle Small Key
     [0x0118] = ITEM_ROW(0x53,      SILVER_CHEST, 0x41, -1, 0x913E, 0x01A2, 0x89, no_upgrade, give_small_key, TCG_ID,    -1, resolve_text_small_keys_cmg), // Small Key (Chest Game)
+
+    // 0x0119 and 0x011A used in other PR
+
+    [0x011B] = ITEM_ROW(0x53,      GILDED_CHEST, 0x0B, -1, 0x004F, 0x00DD, 0x2E, no_upgrade, no_effect, -1, -1, NULL), // Max Upgrade Hookshot
+    [0x011C] = ITEM_ROW(0x53,      GILDED_CHEST, 0x52, -1, 0x005C, 0x012D, 0x4A, no_upgrade, no_effect, -1, -1, NULL), // Max Upgrade Strength
+    [0x011D] = ITEM_ROW(0x53,      GILDED_CHEST, 0x41, -1, 0x90B4, 0x00BF, 0x18, no_upgrade, give_max_upgrade_bomb_bag, -1, -1, NULL), // Max Upgrade Bomb Bag
+    [0x011E] = ITEM_ROW(0x53,      GILDED_CHEST, 0x41, -1, 0x90B5, 0x00E9, 0x35, no_upgrade, give_max_upgrade_bow, -1, -1, NULL), // Max Upgrade Bow
+    [0x011F] = ITEM_ROW(0x53,      GILDED_CHEST, 0x41, -1, 0x90B6, 0x00E7, 0x33, no_upgrade, give_max_upgrade_slingshot, -1, -1, NULL), // Max Upgrade Slingshot
+    [0x0120] = ITEM_ROW(  -1,      GILDED_CHEST,   -1, -1,     -1, 0x00D1, 0x22, max_upgrade_wallet_upgrade, no_effect, 3, -1, NULL), // Max Upgrade Wallet
+    [0x0121] = ITEM_ROW(0x53,      GILDED_CHEST, 0x54, -1, 0x00CE, 0x00DB, 0x2B, no_upgrade, no_effect, -1, -1, NULL), // Max Upgrade Scale
+    [0x0122] = ITEM_ROW(0x53,       BROWN_CHEST, 0x9B, -1, 0x00A8, 0x00BB, 0x12, no_upgrade, no_effect, -1, -1, NULL), // Max Upgrade Nut
+    [0x0123] = ITEM_ROW(0x53,       BROWN_CHEST, 0x99, -1, 0x0091, 0x00C7, 0x1B, no_upgrade, no_effect, -1, -1, NULL), // Max Upgrade Stick
+    [0x0124] = ITEM_ROW(0x53,      GILDED_CHEST, 0x41, -1, 0x9019, 0x0197, 0x7E, no_upgrade, give_bombchus, 20, -1, NULL), // Max Upgrade Bombchu
```

Comment:
Does this need to be a separate item? It seems to be identical in behavior to a pack of 20 chus. It also seems to be missing the upgrade to a chu bag.

Reply:
iirc I added this one for completeness sake and as a placeholder in case the bombchu pack logic changed in the future.
I can remove it if prefered.